halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/ci/jenkins/JenkinsAddMasterCommand.java

```diff
@@ -49,12 +49,19 @@ protected String getCiName() {
   )
   public String password;
 
+  @Parameter(
+      names = "--csrf",
+      description = JenkinsCommandProperties.CSRF_DESCRIPTION
+  )
+  public Boolean csrf;
+
   @Override
   protected Master buildMaster(String masterName) {
     JenkinsMaster master = (JenkinsMaster) new JenkinsMaster().setName(masterName);
     master.setAddress(address)
         .setPassword(password)
-        .setUsername(username);
+        .setUsername(username)
+        .setCsrf(csrf);
```

Comment:
OK if this is null?

Reply:
Yes, it defaults to false in Igor. Should I add a default in hal also?

Reply:
No that's fine then, just checking